docs/backend-system/building-backends/08-migrating.md

```diff
@@ -754,3 +754,255 @@ going easily, but feel free to move it out to where it fits best. As you migrate
 your entire plugin flora to the new backend system, you will probably make more
 and more of these modules as "first class" things, living right next to the
 implementations that they represent, and being exported from there.
+
+### The Auth Plugin
+
+A basic installation of the auth plugin will look as follows.
+
+```ts title="packages/backend/src/index.ts"
+const backend = createBackend();
+/* highlight-add-start */
+backend.add(import('@backstage/plugin-auth-backend'));
+backend.add(import('@backstage/plugin-auth-backend-module-microsoft-provider'));
+/* highlight-add-end */
+```
+
+An additional step you'll need to take is to add the resolvers to your configuration, here's an example:
+
+```yaml title:"app-config.yaml
+auth:
+  environment: development
+  providers:
+    microsoft:
+      development:
+        clientId: ${AZURE_CLIENT_ID}
+        clientSecret: ${AZURE_CLIENT_SECRET}
+        tenantId: ${AZURE_TENANT_ID}
+        signIn:
+          resolvers:
+            - resolver: emailMatchingUserEntityAnnotation
+            - resolver: emailMatchingUserEntityProfileEmail
+            - resolver: emailLocalPartMatchingUserEntityName
```

Comment:
I can verify that it is working after migrating to new backend, without any change to the frontend